### ISSUE TITLE
Status window

### ISSUE THREAD

**Issue by elferherrera** [helix-editor/helix#1971]
Is it possible to add a status window for the editor? 

I just installed it using cargo install and copied the files to the helix folder and I don't know if helix had recognised them. It would be nice to know how many grammar files have been read and from what folder

Also, this status window could provide information about LSP. It could report of it has read the current project, what language it is and what analyser is using

It could work like a status monitor for all the features

**Comment by archseer** [comment]
Try `hx --health`

**Comment by elferherrera** [comment]
Thanks. That shows what I need to know. It isnt loading the highlight syntax files even though it is soft linked to the runtime folder in cargo

**Comment by archseer** [comment]
Try setting the HELIX_RUNTIME env var, point it to the `runtime` folder inside this git repo. 

For example I use `/home/<user>/src/helix/runtime` since that's where I have the repo. Alternatively

```
mkdir ~/.config/helix
ln -s <path to git repo>/runtime ~/.config/helix
```

**Comment by elferherrera** [comment]
I'm running it in windows and created a soft link to the folder in cargo bin. Using the `health` flag I see that the path is all weird. It is mixing back and forward slash

**Comment by elferherrera** [comment]
I see what I did wrong. I'm pointing to the cargo bin and not to the path in git.

**Comment by elferherrera** [comment]
Still. It would be nice to have a window showing the status for lsp

**Comment by MuchHustle** [comment]
this is what has been screwing me up. fucking 3 days wasted on this. build from git clone and build doesnt build it to a git dir, its in .cargo and I don't know enough how to set runtime from wherever it is nofollowedw becAUSE I followed the directions that said from user-name/src/ etc... and that didn't work so I also followed the next "or" option of linking it and --health is a runtime reck with no lsp or tutor or treesitter... I can't even uninstall it.
